tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -310,9 +306,9 @@ - (void)dealloc {
 }
 
 TEST(NSPredicate, Matches) {
-    NSMutableArray* animals = [[[NSMutableArray alloc] init] autorelease];
+    NSMutableArray* animals = [NSMutableArray array];
 
-    PredicateTestAnimal* animal = [[[PredicateTestAnimal alloc] initWithName:@"test1" age:5 speed:115] autorelease];
+    PredicateTestAnimal* animal = [[[PredicateTestAnimal alloc] initWithName:@"test" age:5 speed:115] autorelease];
```

Comment:
this is due to the fact that with created options, when you do the regex match you need an absolute match. We weren't supporting the full array of options, since we didn't have the functionality to map them.
Now there is coreFoundation. I'm going to send out the changes in that CR (for few of the old todos, that can be supported now).
 

---
In reply to: [73245208](https://github.com/Microsoft/WinObjC/pull/788#discussion_r73245208) [](ancestors = 73245208)